sql_generators/active_users_aggregates_v4/__init__.py

```diff
@@ -189,10 +189,68 @@ def generate(target_project, output_dir, use_cloud_function):
             skip_existing=False,
         )
 
+        # Write backfill YAML files.
+        if browser.name == "firefox_desktop":
+            write_sql(
+                output_dir=output_dir,
+                full_table_id=f"{target_project}.{browser.name}_derived.{TABLE_NAME}",
+                basename="backfill.yaml",
+                sql=render(
+                    desktop_backfill_template,
+                    template_folder=THIS_PATH / "templates",
+                    format=False,
+                ),
+                skip_existing=False,
+            )
+
+        # Write checks sql files.
         write_sql(
             output_dir=output_dir,
             full_table_id=f"{target_project}.{browser.name}_derived.{TABLE_NAME}",
             basename="checks.sql",
             sql=checks_sql,
             skip_existing=False,
         )
+
+        if browser.name == "focus_android":
+            write_sql(
+                output_dir=output_dir,
+                full_table_id=f"{target_project}.{browser.name}.{BASE_NAME}",
+                basename="view.sql",
+                sql=reformat(
+                    focus_android_view_template.render(
+                        project_id=target_project,
+                        app_name=browser.name,
+                        table_name=TABLE_NAME,
+                    )
+                ),
+                skip_existing=False,
+            )
+        elif browser.name == "firefox_desktop":
+            write_sql(
+                output_dir=output_dir,
+                full_table_id=f"{target_project}.{browser.name}.{BASE_NAME}",
+                basename="view.sql",
+                sql=reformat(
+                    view_template.render(
+                        project_id=target_project,
+                        app_name=browser.name,
+                        table_name=TABLE_NAME,
+                    )
+                ),
+                skip_existing=False,
+            )
+        else:
+            write_sql(
+                output_dir=output_dir,
+                full_table_id=f"{target_project}.{browser.name}.{BASE_NAME}",
+                basename="view.sql",
+                sql=reformat(
+                    view_template.render(
+                        project_id=target_project,
+                        app_name=browser.name,
+                        table_name=TABLE_NAME,
+                    )
+                ),
+                skip_existing=False,
+            )
```

Comment:
These two conditions seem to be identical... is that intentional? If so, could you remove the `elif` and just use the `else`?